cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/hbase/HBaseStreamConsumerStateStoreFactory.java

```diff
@@ -52,25 +51,24 @@
   @Override
   public synchronized StreamConsumerStateStore create(StreamConfig streamConfig) throws IOException {
     Id.Namespace namespace = streamConfig.getStreamId().getNamespace();
-    byte[] tableName = Bytes.toBytes(getTableName(namespace));
-
+    TableId streamStateStoreTableId = TableId.from(getTableName(namespace));
```

Comment:
Yeah, this should again go away when we can have the table name and the namespace Id separate.